lib/active_shipping/carriers/fedex.rb

```diff
@@ -619,20 +619,17 @@ def parse_tracking_response(response, options)
 
         tracking_number = tracking_details.at('TrackingNumber').text
         status_detail = tracking_details.at('StatusDetail')
-        if status_detail.nil?
-          raise ActiveShipping::Error, "Tracking response does not contain status information"
-        end
-
-        status_code = status_detail.at('Code').try(:text)
-        if status_code.nil?
-          raise ActiveShipping::Error, "Tracking response does not contain status code"
-        end
+        if status_detail.blank?
+          status_code, status, status_description, delivery_signature = nil
```

Comment:
Ultimately not necessary since variables defined within a conditional are `nil`-initialized if the conditional wasn't true, but I thought it'd be nice to make it clear exactly what variables would be `nil` as a result of `status_detail` being blank.